rules/framework/framework_packaging.py

```diff
@@ -108,6 +108,8 @@ def main():
             lambda args: _merge_binaries(args.framework_root, args.framework_name, args.inputs),
         "modulemap":
             lambda args: _copy_modulemap(args.framework_root, args.input()),
+        "plist":
+            lambda args: _cp(args.input(), args.output()),
```

Comment:
Since a framework bundle could contain other `*.plist` resources, can we name this explicitly `infoplist` action?